README.md

```diff
@@ -17,61 +17,63 @@ git init
 ````
 Vous allez maintenant lier votre répertoire ````week-3```` situé sur votre ordinateur avec un répertoire distant week-3 situé sur votre compte github. Le lien sera appelé ````origin````.
 Pour cela, créez un nouveau répertoire appelé ````week-3```` sur Github et copiez l'url de ce répertoire.
-Puis, faites la commande suivante, en remplaçant l'url par la votre:
+Puis, faites la commande suivante, en remplaçant (votre compte) dans cette adresse par votre compte :
 ``` Console
 git remote add origin git@github.com:(votre compte)/week-3.git
 ````
 Cela vous permet de synchroniser votre compte github avec les modifications que vous ferez sur le projet week-3 sur votre ordinateur.
 
-A cette étape, si vous faites ````ls```` dans votre console, le dossier ````week-3```` doit etre vide.
+À cette étape, si vous faites ````ls```` dans votre console, le dossier ````week-3```` doit être vide.
+Et si vous faites ````ls -a```` le dossier ````week-3```` ne contient que les fichiers de configuration de git, dans le dossier caché ````.git````.
```

Comment:
We might have to add a link for ls command explaination, as it's a new notion for some people